Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs

```diff
@@ -1,149 +1,135 @@
-		{
-			SetDarkLightToggleInitialState();
-		}
-
-		private void SetDarkLightToggleInitialState()
-		{
-			// Initialize the toggle to the current theme.
-			var root = global::Windows.UI.Xaml.Window.Current.Content as FrameworkElement;
-
-			switch (root.ActualTheme)
-			{
-				case ElementTheme.Default:
-					DarkLightModeToggle.IsChecked = SystemThemeHelper.GetSystemApplicationTheme() == ApplicationTheme.Dark;
-					break;
-				case ElementTheme.Light:
-					DarkLightModeToggle.IsChecked = false;
-					break;
-				case ElementTheme.Dark:
-					DarkLightModeToggle.IsChecked = true;
-					break;
-			}
-		}
-
-		/// <summary>
-		/// This method handles the top padding for phones like iPhone X.
-		/// </summary>
-		private void InitializeSafeArea()
-		{
-			var full = Windows.UI.Xaml.Window.Current.Bounds;
-			var bounds = ApplicationView.GetForCurrentView().VisibleBounds;
-
-			var topPadding = Math.Abs(full.Top - bounds.Top);
-
-			if (topPadding > 0)
-			{
-				TopPaddingRow.Height = new GridLength(topPadding);
-			}
-		}
-
-		private void ToggleButton_Click(object sender, RoutedEventArgs e)
-		{
-			// Set theme for window root.
-			if (global::Windows.UI.Xaml.Window.Current.Content is FrameworkElement root)
-			{
-				switch (root.ActualTheme)
-				{
-					case ElementTheme.Default:
-						if(SystemThemeHelper.GetSystemApplicationTheme() == ApplicationTheme.Dark)
-						{
-							root.RequestedTheme = ElementTheme.Light;
-						}
-						else
-						{
-							root.RequestedTheme = ElementTheme.Dark;
-						}
-						break;
-					case ElementTheme.Light:
-						root.RequestedTheme = ElementTheme.Dark;
-						break;
-					case ElementTheme.Dark:
-						root.RequestedTheme = ElementTheme.Light;
-						break;
-				}
-			}
-		}
-
-		private void OnNestedSampleFrameChanged(DependencyObject sender, DependencyProperty dp)
-		{
-			var isInsideNestedSample = NestedSampleFrame.Content != null;
-
-			// prevent empty frame from blocking the content (nav-view) behind it
-			NestedSampleFrame.Visibility = isInsideNestedSample
-				? Visibility.Visible
-				: Visibility.Collapsed;
-
-			// toggle built-in back button for wasm (from browser) and uwp (on title bar)
-			SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = isInsideNestedSample
-				? AppViewBackButtonVisibility.Visible
-				: AppViewBackButtonVisibility.Collapsed;
-		}
-
-		public void ShowNestedSample<TPage>(bool? clearStack = null) where TPage : Page
-		{
-			var wasFrameEmpty = NestedSampleFrame.Content == null;
-			if (NestedSampleFrame.Navigate(typeof(TPage)) && (clearStack ?? wasFrameEmpty))
-			{
-				NestedSampleFrame.BackStack.Clear();
-			}
-		}
-
-		public bool BackNavigateFromNestedSample()
-		{
-			if (NestedSampleFrame.Content == null)
-			{
-				return false;
-			}
-
-			if (NestedSampleFrame.CanGoBack)
-			{
-				NestedSampleFrame.GoBack();
-			}
-			else
-			{
-				NestedSampleFrame.Content = null;
+    public sealed partial class Shell : UserControl
```

Comment:
might want to fix the space/tab settings, you just replaced all tabs by spaces

Reply:
oh! Not sure why it does not apply project defaults

Reply:
Fixed now

Reply:
because there is no `.editorconfig` in this project